pkg/watcher/crd/watcher.go

```diff
@@ -143,6 +141,7 @@ func updateTracingPolicy(ctx context.Context, log logrus.FieldLogger, s *sensors
 			"new": newTp.TpName(),
 		}).Info("updating tracing policy")
 		update(oldTp, newTp)
+		return
```

Comment:
hmm rather than putting more returns here maybe we could have update return an error and then if (err) ... print the warning?

Reply:
Yes, this may be a cleaner solution there. I will try to make that better.

Reply:
Updated that a bit. The error is not due to the update but on the different types of tracing policies. I believe it is cleaner now.

Reply:
ah that is much clearer to me what the issue was. thanks.